server.js

```diff
@@ -35,7 +35,7 @@ requirejs.config({
 var logger = module.exports = new(winston.Logger)({
     transports: [
         new(winston.transports.Console)({
-            colorize: true,
+            colorize: false,
```

Comment:
I like it colorized for normal viewing... maybe should configure Winston to log to a different file by default as well, which it can do natively.  Move aside to another issue, remove from this PR.